app/components/markdown/markdown_table/index.js

```diff
@@ -7,12 +7,15 @@ import {connect} from 'react-redux';
 import {getTheme} from 'mattermost-redux/selectors/entities/preferences';
 
 import {goToScreen} from 'app/actions/navigation';
+import {isTablet, isLandscape} from 'app/selectors/device';
```

Comment:
there is a more accurate way to detect if the device is a tablet

`IS_TABLET` from DeviceTypes constants.

Also I would try and avoid isLandscape and use `Dimensions` from ReactNative to determine if it is landscape or not.

Let me know if you need more details.

Reply:
Ah i wasn't aware of these two methods, seems like a simple enough fix :)

Will change it right away